python/ray/air/execution/resources/request.py

```diff
@@ -237,7 +237,7 @@ def annotate_remote_entities(
             # The empty head bundle is place on the first bundle index with empty
             # resources.
             annotated.append(
-                self._annotate_remote_entity(entities[0], {}, bundle_index=0)
+                self._annotate_remote_entity(entities[0], {"CPU": 1}, bundle_index=0)
```

Comment:
This is tricky. If the resources is empty, that actor can be scheduled on any node.